ipam/ipam.go

```diff
@@ -162,17 +162,17 @@ func (ipam *IPAM) allocateIP(input interface{}, ctx common.RestContext) (interfa
 		}
 
 		if found {
-			endpoint.TenantId = fmt.Sprintf("%d", t.ID)
-			log.Printf("IPAM: Tenant '%s' has ID %s, original %d", tenantParam, endpoint.TenantId, t.ID)
+			endpoint.TenantId = t.ID
```

Comment:
`golint` is the style checker. I hadn't changed the existing names or inconsistent ID vs Id in this PR, only the types and tags. Happy to fix it now rather than later though.